pkg/minikube/registry/drvs/hyperkit/hyperkit.go

```diff
@@ -37,6 +39,11 @@ const (
 	docURL = "https://minikube.sigs.k8s.io/docs/reference/drivers/hyperkit/"
 )
 
+var (
+	// hyperkitSpecificVersion is used by hyperkit versionCheck(whether user's hyperkit is older than this)
+	hyperkitSpecificVersion = "0.20190201"
```

Comment:
Instead of `hyperkit.hyperkitSpecificVersion` - can this be renamed to `minimumVersion`?

Reply:
Thanks nice naming!
I renamed.